### ISSUE TITLE
[Bug] Wave cap of 1 still shows wave action with high creeper flow

### ISSUE THREAD

**Issue by TrickyCorp** [KnuckleCracker/CW4-bug-tracker#693]
Context: wave cap is 1 and creeper flow is exactly 100.

There is wave behavoir. Creeper numbers go up and down in a pool even if I stopped shooting at it, just like how waves work.

Also, creeper doesn't wanna flow off high platforms even with these settings (on a different part of the level)
It does flow, but creeper just keeps accumulating on the edge instead of all falling down.

I'm not sure if this is a bug or intended?

I'll add some screenshots as a context.

**Comment by TrickyCorp** [comment]
Look at these 4 screenshots of an area that was just hit by a mortar (I checked this again to be sure)
Pretty sure this proves there's wave action instead of CW3-ish creeper flow.
Summary: after being hit my the mortar shell, the creeper is somehow higher that the surrounding areas even though it used to be around the same height.
Again, not sure if this is intended behaviour, but it doesn't feel like it.

![Schermopname (21)](https://user-images.githubusercontent.com/75793635/106379802-cece0680-63ae-11eb-95d0-d4087f6afa55.png)

![Schermopname (22)](https://user-images.githubusercontent.com/75793635/106379808-d42b5100-63ae-11eb-9d3e-ffa23b29d9ad.png)

![Schermopname (23)](https://user-images.githubusercontent.com/75793635/106379814-d7264180-63ae-11eb-9198-17f7db84c621.png)

![Schermopname (24)](https://user-images.githubusercontent.com/75793635/106379817-da213200-63ae-11eb-85e9-31747aa9c3b7.png)

Sorry for the massive wall of screenshots by the way.

**Comment by TrickyCorp** [comment]
And here is a screenshot showing creeper that doesn't flow down despite the wave cap of 1 and 100 creeper flow.
![Schermopname (25)](https://user-images.githubusercontent.com/75793635/106380135-8fa0b500-63b0-11eb-9bf0-a84099210c95.png)

This should contain the save files. I sure hope I did it right.
[643.zip](https://github.com/KnuckleCracker/CW4-bug-tracker/files/5899465/643.zip)

I hope I did things right this time, providing information about the issue etc.
